app/src/debug/java/com/futurice/freesound/app/module/InstrumentationModule.java

```diff
@@ -40,12 +40,11 @@
     @Provides
     @Singleton
     @ApiModule.NetworkInterceptors
-    static List<Interceptor> provideNetworkInterceptors(FreeSoundApiInterceptor apiInterceptor,
-                                                        HttpLoggingInterceptor loggingInterceptor) {
+    static List<Interceptor> provideNetworkInterceptors(HttpLoggingInterceptor loggingInterceptor,
+                                                        StethoInterceptor stethoInterceptor) {
         List<Interceptor> networkInterceptors = new ArrayList<>(1);
```

Comment:
It's worth initializing the size to 2 rather than 1, because we immediately add 2 items here.